### DISCUSSION TITLE
Csla.Blazor.IPropertyInfo - a couple of things.

### DISCUSSION THREAD

**Discussion by j055** [MarimerLLC/csla#1972] [Ideas]
Just playing around with the `ViewModel` in Blazor. I notice that the implementation of `Csla.Blazor.IPropertyInfo.ErrorText` joins multiple messages with a ',' but needs a space after the comma to display properly. 

Having said that I would prefer if `Csla.Blazor.IPropertyInfo.ErrorText` and it's siblings returned a collection of strings. An implicit `ToString` could join them with a comma/space.

It would be nice to have a `Name` property in `IPropertyInfo` so it can easily be used as an `id` in HTML elements. 

Minor things really but thought I'd mention them. 

Thank you

**Comment by rockfordlhotka** [comment]
That's good feedback @j055, thank you!

I've added a couple issues to the backlog to track these ideas.

The missing space is a bug. The API change is breaking, so would need to wait for v6.0
